docs/content/templates/functions.md

```diff
@@ -214,6 +214,23 @@ e.g.
     </ul>
 
 
+### union
+Given two arrays (or slices) A and B, this function will return a new array that contains the elements or objects that belong to either A or to B or to both. The elements supported are strings, integers and floats (only float64).
```

Comment:
Good point. Then the intersection function should support "slices of structs and pointers" type as well.